src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3355,8 +3355,10 @@ private BoundNode BindConstructorBody(ConstructorDeclarationSyntax constructor,
             Debug.Assert(bodyBinder != null);
 
             if (constructor.Initializer?.IsKind(SyntaxKind.ThisConstructorInitializer) != true &&
-                ContainingType.GetMembersUnordered().OfType<SynthesizedRecordConstructor>().Any())
+                ContainingType.GetMembersUnordered().OfType<SynthesizedRecordConstructor>().Any() &&
+                !SynthesizedRecordCopyCtor.FindCopyConstructor(ContainingType).Equals(this.ContainingMember(), TypeCompareKind.ConsiderEverything))
```

Comment:
>!SynthesizedRecordCopyCtor.FindCopyConstructor(ContainingType).Equals(this.ContainingMember(), TypeCompareKind.ConsiderEverything) [](start = 16, length = 130)

Why so complicated? Can't we simply check the signature of the containing method?